### ISSUE TITLE
Android Studio proguard warnning kotlinx.atomicfu.AtomicFU: can't find referenced method 'void setInterceptor

### ISSUE THREAD

**Issue by wuseal** [Kotlin/kotlinx.atomicfu#57]
When I use Coroutine 1.2.0-alpha, after run proguard, This warming was throw out, I don't known if I can use `-dontwarn` to make it disappear, Wheather it could broken Android APP later?
```
Warning: kotlinx.atomicfu.AtomicFU: can't find referenced method 'void setInterceptor(kotlinx.atomicfu.AtomicOperationInterceptor)' in program class kotlinx.atomicfu.InterceptorKt
```
![image](https://user-images.githubusercontent.com/9211902/55390658-3ecc9500-556a-11e9-8398-f7413c6d5b7b.png)

**Comment by jemshit** [comment]
Any update ? With latest coroutine and kotlin version this forces build to fail

**Comment by elizarov** [comment]
Fixed in `0.12.7`
